lib/rubocop/cop/style/space_after_colon.rb

```diff
@@ -12,21 +12,35 @@ class SpaceAfterColon < Cop
 
         def on_pair(node)
           oper = node.loc.operator
-          return unless oper.is?(':') && followed_by_space?(oper)
 
-          add_offense(oper, oper)
+          return unless oper.is?(':')
+
+          add_offence_if_colon_not_followed_by_space(oper)
         end
 
         def on_if(node)
           return unless ternary_op?(node)
 
-          colon = node.loc.colon
-          return unless followed_by_space?(colon)
+          add_offence_if_colon_not_followed_by_space(node.loc.colon)
+        end
+
+        def on_kwoptarg(node)
+          name = node.loc.name
+          # The name source range is inclusive of the variable name and colon.
+          colon = Parser::Source::Range.new(
```

Comment:
Good point about naming! This has got me thinking:

The `colon` source range is used in two places: 
1. `not_followed_by_space?` to determine if the cop should add an offen**s**e (:smile:)
2. In the code that generates the cop report  (the offending source is underlined)

I'll have to test it this evening, but I think 1. would work fine with the range copying/extending as you suggest, but 2. would highlight the whole name including the colon, rather than just the colon as my current PR does.

i.e. 
```
/tmp/bar.rb:1:15: C: Space missing after colon.
def f(some_arg:1)
              ^
```
vs
```
/tmp/bar.rb:1:7: C: Space missing after colon.
def f(some_arg:1)
      ^^^^^^^^^
```

I think I prefer the latter, as it's more precise (so I would leave the current "custom" `Range` construction, rather than change to use `resize`), what do you think?